### PR TITLE
Original hill function definition

### PR DESCRIPTION
**Pull request by juanitorduz**
Closes https://github.com/pymc-labs/pymc-marketing/issues/917

In https://github.com/pymc-labs/pymc-marketing/issues/917#issuecomment-2284338338 it is suggested to use Equation 5 from https://static.googleusercontent.com/media/research.google.com/en//pubs/archive/46001.pdf to have the original definition. I renamed the previous one as people might find it useful.

----
📚 Documentation preview 📚: https://pymc-marketing--925.org.readthedocs.build/en/925/

### CODE REVIEW

**Comment by juanitorduz** [comment]
@wd60622 What do you think about this proposal? I think it is very reasonable.

File: pymc_marketing/mmm/delayed_saturated_mmm.py [hunk 1]
```diff
@@ -63,11 +63,11 @@
 
 class BaseMMM(BaseValidateMMM):
     """
-    Base class for a media mix model using Delayed Adstock and Logistic Saturation (see [1]_).
+     Base class for a media mix model using Delayed Adstock and Logistic Saturation (see [1]_).
 
     References
-    ----------
-    .. [1] Jin, Yuxue, et al. “Bayesian methods for media mix modeling with carryover and shape effects.” (2017).
+     ----------
```
Comment (wd60622) [inline pymc_marketing/mmm/delayed_saturated_mmm.py:69]: what's up this this new space?

**Review by wd60622 — APPROVED** [review]
Looks good. Just question about the spaces. Don't think that causes any issues with docs though

File: pymc_marketing/mmm/delayed_saturated_mmm.py (hunk 1 above)
Comment (juanitorduz) [reply pymc_marketing/mmm/delayed_saturated_mmm.py]: super weird! fixed!

**Comment by juanitorduz** [comment]
> Looks good. Just question about the spaces. Don't think that causes any issues with docs though

Thanks! Good catch! Fixed!

**Pull request merged**